File: C++/Daily Flash/week1/Day3_Jan15/code_C++/program1.cpp
/*
 *
Program 1: Write a Program to check whether the Character is Uppercase or Lowercase Character.
Input: v
Output: letter v is in lowercase
*
*/

#include <iostream>

int main() {
	char c;
	std::cout << "Enter Character: ";
	std::cin >> c;
	
	if( (c >=97 && c<=122) || (c >=65 && c <=90) ) {
			
			if(c >=97 && c<=122)
				std::cout << "letter " << c  << " is in lower case" <<std::endl;
			else
				std::cout << "letter " << c  << " is in upper case" <<std::endl;
	}
	else  
			std::cout << "letter " << c  << " is not an alphabet" <<std::endl;
}

File: PYTHON/Daily Flash/week1/Day3_Jan15/code_python/program1.py
'''
Program 1: Write a Program to check whether the Character is Uppercase or
Lowercase Character.
Input: v
Output: letter v is in lowercase
'''

char = input("Enter character: ")

#print(ord(char))

if( (ord(char) >= 65 and ord(char) <=90) or (ord(char) >=97 and ord(char)<=122 ) ):
        
        if((ord(char) >= 65 and ord(char) <=90) ):
            print("Letter " , char , " is in Uppercase")
        else:
            print("Letter " , char , " is in Lowercase")

else:
            print("Letter " , char , " is not an alphabet")


